### DISCUSSION TITLE
PDF Annotation

### DISCUSSION THREAD

**Discussion by CobriMediaJulien** [TriliumNext/Trilium#4884] [Ideas] [answered]
Hey guys,

did anybody allrdy look into this?`Do we have proof of concepts or ideas/ protoypes how to implement sth like this?

**Comment by meichthys** [comment, marked as answer]
Being Tracked in TriliumNext here: https://github.com/TriliumNext/Notes/issues/76

**Comment by eliandoran** [comment]
This is now possible using with the new PDF.js handling that will appear in the upcoming version.
See https://docs.triliumnotes.org/user-guide/concepts/notes/printing-and-pdf-export.

**Reply by CobriMediaJulien** [reply]
Wow! That us awesome!
